flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/StandaloneAutoscalerEntrypoint.java

```diff
@@ -60,9 +60,11 @@ public static <KEY, Context extends JobAutoScalerContext<KEY>> void main(String[
 
         var autoScaler = createJobAutoscaler(eventHandler, stateStore);
 
-        var autoscalerExecutor =
-                new StandaloneAutoscalerExecutor<>(conf, jobListFetcher, eventHandler, autoScaler);
-        autoscalerExecutor.start();
+        try (var autoscalerExecutor =
+                new StandaloneAutoscalerExecutor<>(
+                        conf, jobListFetcher, eventHandler, autoScaler)) {
+            autoscalerExecutor.start();
+        }
```

Comment:
Hey @plugatarev , thanks for the PR!

IIUC, we cannot close `StandaloneAutoscalerExecutor`. We expect StandaloneAutoscalerExecutor scaling jobs periodically. It will stop work if we close executor.

Reply:
Yes, you are right